src/utils/components/ErrorWindow.vue

```diff
@@ -35,38 +35,36 @@ const emit = defineEmits(['close'])
 </script>
 
 <template>
-    <teleport to="#main-component">
```

Comment:
@ltshb it looks like the additional div was necessary due to the 'in-place' teleport of this component, is there any reason to keep this teleport around or can it be deleted since this component is not used anywhere else?